packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -536,7 +536,10 @@ class OffsetLayer extends ContainerLayer {
 
   @override
   Object findRegion(Offset offset, Type type) {
-    return super.findRegion(offset + this.offset, type);
```

Comment:
Please name the argument something that doesn't shadow a member variable.

Reply:
Fixed